build.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
 
 group = 'org.finos.symphony.wdk'
 
-ext.projectVersion = '0.0.6-SNAPSHOT'
+ext.projectVersion = '1.1.0-RC'
```

Comment:
can we keep it a 1.0.0.-RCXX to avoid burning the 1.0.0 version?

Reply:
done